enterprise/app/trends/heatmap.tsx

```diff
@@ -226,6 +227,13 @@ class HeatmapComponentInternal extends React.Component<HeatmapProps, State> {
     }
   }
 
+  maybeFireZoomCallback() {
+    const s = this.computeHeatmapSelection();
```

Comment:
nit: the single-letter naming convention is more of a go-ism

Reply:
fair enough, replaced this and the one above it, too.